lib/fuzion/std/fileio.fz

```diff
@@ -0,0 +1,53 @@
+# This file is part of the Fuzion language implementation.
+#
+# The Fuzion language implementation is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, version 3 of the License.
+#
+# The Fuzion language implementation is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of Fuzion standard library feature fuzion.std.fileio
+#
+#
+# -----------------------------------------------------------------------
+
+# fuzion.std.fileio -- fileio presents two simple methods:
+# readFile to retrieve data from a file
+# WriteFile to insert data into a file 
+#
+public fuzion.std.fileio is
+
+  # read a file into an array u8.
+  # 
+  # NYI : an equivalent to a Java MappedByteBuffer would be a better solution
```

Comment:
this NYI comment is not need, MappedByteBuffer is a different solution that is better for some uses, worse for others.